src/ext/java/org/opentripplanner/ext/bikerentalservicedirectory/BikeRentalServiceDirectoryFetcher.java

```diff
@@ -53,7 +54,14 @@
       }
     }
     catch (java.io.IOException e) {
-      LOG.warn("Error fetching list of bike rental endpoints from {}", url, e);
+      LOG.warn("Error fetching list of bike rental endpoints from {}", parameters.getUrl(), e);
+    }
+    catch (NullPointerException e) {
+      LOG.warn(
+          "Error reading json from {}. Are json tag names configured properly?",
+          parameters.getUrl(),
+          e
+      );
```

Comment:
A NPE is a "system level" (low level) exception that you should never catch in this way. Use a `if x==null, then warn and exit` instead.